### ISSUE TITLE
`fileInputUpdatePath` stopped applying to exisiting file inputs (introduced in 0.41.0)

### ISSUE THREAD

**Issue by yardensachs** [unfoldadmin/django-unfold#855]
*Versions:* Django 5 + Unfold 0.41 + latest Chrome

**Are you able to replicate the bug in the demo site?**
It looks like no. I am not sure that the demo site has 0.41.0 yet

## Description
There's an issue with the `fileInputUpdatePath` implementation that prevents it from working with file inputs that exist when the page initially loads. This was introduced in #846.

Currently, the MutationObserver only watches for newly added elements (`childList` mutations) and attaches the change event handler to file inputs after they're dynamically added to the DOM. However, this means any file inputs that are part of the initial page HTML are missed completely.

### Suggested Fix

We should modify the function to:
1. Immediately attach handlers to existing file inputs when the function is called
2. Keep the MutationObserver for handling dynamically added inputs

Here's how the code could be restructured:
```javascript
const fileInputUpdatePath = () => {
  const handleFileInput = (input) => {
    input.addEventListener("change", (e) => {
      const parts = e.target.value.split("\\");
      const placeholder = input.parentNode.parentNode.parentNode.querySelector(
        "input[type=text]"
      );
      placeholder.setAttribute("value", parts[parts.length - 1]);
    });
  };

  // Handle existing inputs
  document.querySelectorAll("input[type=file]").forEach(handleFileInput);

  // Watch for new inputs
  const observer = new MutationObserver((mutations) => {
    for (const mutation of mutations) {
      if (mutation.type === "childList") {
        document.querySelectorAll("input[type=file]").forEach(handleFileInput);
      }
    }
  });

  observer.observe(document.body, {
    childList: true,
    subtree: true,
  });
};
```

**Comment by yardensachs** [comment]
Addressed in #856

**Comment by lukasvinclav** [comment]
@yardensachs I already did some testing and I was not able to replicate the issue. When I checked the JS code, I saw event handler is mounted immediately on the first load. I even added new imagefield to the inlines to be able to see other use cases. Here is the demo example:

https://demo.unfoldadmin.com/en/admin/formula/driver/56/change/

**Comment by yardensachs** [comment]
@lukasvinclav I clicked on the link and was able to reproduce. Here is the video:

https://github.com/user-attachments/assets/468f1c4a-3011-43ed-9ec1-267499ae9c5f

**Comment by lukasvinclav** [comment]
Now, I can confirm the issue. It doesn't work in Chrome. Firefox seems to be okay.

**Comment by lukasvinclav** [comment]
Fixed in #861
